shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -87,7 +89,28 @@
           .hasArg(true)
           .numberOfArgs(1)
           .argName("host-file")
-          .desc("Host File contains worker hosts, each line has a worker host")
+          .desc("Host File contains worker hosts, each line has a worker host, if empty,"
+              + " all hosts will be selected unless explicitly excluded by setting excluded-hosts")
+          .build();
+  private static final Option EXCLUDED_HOSTS_OPTION =
+      Option.builder()
+          .longOpt("excluded-hosts")
+          .required(false)
+          .hasArg(true)
+          .numberOfArgs(1)
+          .argName("excluded-hosts")
+          .desc("A list of excluded worker hosts separated by comma,"
+              + " it should not be set together with hosts")
+          .build();
+  private static final Option EXCLUDED_HOST_FILE_OPTION =
+      Option.builder()
+          .longOpt("excluded-host-file")
+          .required(false)
+          .hasArg(true)
+          .numberOfArgs(1)
+          .argName("excluded-host-file")
+          .desc("Host File contains excluded worker hosts, each line has a worker host,"
```

Comment:
has a worker host. Only one of the 'hosts', 'host-file', 'excluded-hosts', and 'excluded-host-file' should be set.